views/partials/data-search-form.njk

```diff
@@ -1,7 +1,7 @@
 <!-- Form -->
 <form action="/search" method="GET">
   <div class="relative mb-10">
-    <input type="text" class="border-2 border-solid border-gray-200 py-4 pl-4 pr-20 text-xl outline-none focus:border-secondary w-full" placeholder="{{__('Search datasets')}}" name="q" value="{{ query.q }}" autofocus>
```

Comment:
I think it breaks the functionality, eg, now if you type a keyword in the searchbox and hit enter, you wouldn't see that keywrod after page reload.

Reply:
@gavram could you reply here please?

Reply:
This really changes the functionality. But in order to fulfill the client's requirement and retain functionality, there are two things that need to be done.
1. that only the search term is retained in the `search input box` and
2. to make search filter badges like the old CKAN portal below `search input box`
The first option requires a change in `frontend-v2` and little change in `theme`.
It takes me a while to clear up some things here. May be one day.

Reply:
This really changes the functionality. But in order to fulfill the client's requirement and retain functionality, there are two things that need to be done.
1. that only the search term is retained in the `search input box` and
2. to make search filter badges like the old CKAN portal below `search input box`
The first option requires a change in `frontend-v2` and little change in `theme`.
It takes me a while to clear up some things here. May be one day.